modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/util/v1_0/SkuUtil.java

```diff
@@ -59,6 +61,10 @@ public static CPInstance addOrUpdateCPInstance(
 		long replacementCProductId = 0;
 		String replacementCPInstanceUuid = null;
 
+		int discontinuedDateMonth = 0;
```

Comment:
Please, move this fields close the their if block

Reply:
@andrea-ale-sbarra, closer to line 99 you mean? They need to stay outside of that if cause they need to be the default values. If they are inside the `if (GetterUtil.getBoolean(sku.getDiscontinued())) {` check, then they wouldn't be initiated if the inserted product is not discontinued.